<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyDialog</class>
 <widget class="QDialog" name="StudyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Experiment manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget::tab-bar { min-width: 10000px;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="buttonsStackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="studiesButtons">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="deleteStudyButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Delete study</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="activateStudyButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Activate study</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startStudyButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Start study</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pauseStudyButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Pause study</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="syncStudyButton">
           <property name="text">
            <string>Sync</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="securityButtons">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="exportPublicKeyToFileButton">
           <property name="text">
            <string>Export to file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sharePublicKeyButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Share</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="studiesTab">
      <attribute name="title">
       <string>Studies</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Study</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="studiesComboBox">
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newStudyButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>New...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="studiesStackedWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="studyInfoPage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Experiments</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QTreeView" name="experimentsTreeView"/>
                </item>
                <item>
                 <widget class="QGroupBox" name="activeStudyStatusMessageGroupBox">
                  <property name="title">
                   <string>Status Message</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <item>
                    <widget class="QTextBrowser" name="activeStudyStatusTextBrowser">
                     <property name="styleSheet">
                      <string notr="true">QTextBrowser{ background-color: transparent; }</string>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::Box</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Plain</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="linkStudyPage">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="linkStudyGroupBox">
             <property name="title">
              <string>Add Study</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Url</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="experimentsUrlEdit">
                <property name="placeholderText">
                 <string>git@github.com:MyUser/mystudy.git</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Branch</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="experimentsBranchComboBox">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="5" column="1">
               <widget class="QGroupBox" name="linkStudyAdvancedForm">
                <property name="title">
                 <string>Advanced</string>
                </property>
                <layout class="QFormLayout" name="formLayout_2">
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="studyNameEdit">
                   <property name="inputMask">
                    <string/>
                   </property>
                   <property name="placeholderText">
                    <string>My new study</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Results url</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="resultsUrlEdit">
                   <property name="placeholderText">
                    <string>git@github.com:MyUser/mystudy.git</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Branch</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="resultsBranchEdit"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Status</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QTextBrowser" name="linkStudyTextBrowser">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">QTextBrowser { background-color: transparent; }</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <widget class="QStackedWidget" name="linkStudyStackedWidget">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="linkStudyButtonPage">
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QPushButton" name="linkStudyButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Link study</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="linkStudyProgressPage">
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QProgressBar" name="linkStudyProgressBar">
                    <property name="maximum">
                     <number>0</number>
                    </property>
                    <property name="value">
                     <number>-1</number>
                    </property>
                    <property name="textVisible">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QCheckBox" name="uploadResultsCheckBox">
                <property name="text">
                 <string>Upload results</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="secureShellKeysTab">
      <attribute name="title">
       <string>Secure Shell keys</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Public key</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="publicKeyTextEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
